src/software/thunderscope/thunderscope_config.py

```diff
@@ -487,11 +488,15 @@ def configure_replay_view(
                     replay_log=blue_replay_log,
                     visualization_buffer_size=visualization_buffer_size,
                     extra_widgets=[],
+                    frame_swap_counter=FrameTimeCounter(),
+                    refresh_counter=blue_refresh_func_counter,
```

Comment:
I need to give some context: 

This is not an anonymous object object because the same counter need to be used in both the `confiure_replay_view` and the `TScopeTab` widget. If we initialize two different object, then we run the trouble of displaying the wrong data.